apps/SageAccountWeb/src/components/TwoFactorAuth/ResetTwoFactorAuth.tsx

```diff
@@ -122,7 +150,6 @@ export function ResetTwoFactorAuth() {
                       }}
                     />
                   )}
-                  <SourceAppLogo />
```

Comment:
The other rendered `<SourceAppLogo />` is shown in this case, so this is a duplicate